api/linkAccount.js

```diff
@@ -0,0 +1,36 @@
+const config = require('../config');
+const message = config.message;
+const helper = require('../helper');
+const responseHelper = helper.response;
+const service = require('../service');
+const _ = require('lodash');
+
+let init = (app) => {
+    app.post('/link-account', linkAccount);
+};
+
+let linkAccount = (request, response) => {
+    const token = request.body;
```

Comment:
Always prefer to pass an object in the body, so we can add more fields in the future without much refactoring. 
{token:'fgewrgerg'}

If we make it into an object, we can use es6 destructuring as well. https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

So 
```
const token = request.body.token
```
becomes:

```
const {token} = request.body;
```